src/Particular.ThroughputCollector.Persistence/PersistenceManifest.cs

```diff
@@ -0,0 +1,130 @@
+﻿namespace Particular.ThroughputCollector.Persistence;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using NServiceBus.Logging;
```

Comment:
I don't think that we should be referencing nservicebus - I've been using the microsoft.extensions.logging everywhere.